benchmarks/pubmedqa/pubmedqa.py

```diff
@@ -0,0 +1,58 @@
+"""
+PubMedQA: A Dataset for Biomedical Research Question Answering
+
+# inspect eval pubmedqa/pubmedqa.py
+"""
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.scorer import choice
+from inspect_ai.solver import multiple_choice
+
+TEMPLATE = r"""
+The following are multiple choice questions (with answers) about medical knowledge.
+
+Answer the following question given the context.
+The entire content of your response should be of the following format: 'ANSWER: $LETTER' (without quotes) where LETTER is one of {letters}.
+
+{question}
+
+{choices}
+""".strip()
+
+
+@task
+def pubmedqa():
+    dataset = hf_dataset(
+        path="bigbio/pubmed_qa",
+        # Note: Each of 10 folds has a different `train` and `validation` set,
+        # but all 10 have the same `test` set.
+        name="pubmed_qa_labeled_fold0_bigbio_qa",
+        sample_fields=record_to_sample,
+        trust=True,
+        split="test",
+    )
+
+    return Task(
+        dataset=dataset,
+        plan=[multiple_choice(template=TEMPLATE)],
+        scorer=choice(),
+    )
+
+
+choices = {
```

Comment:
I would define this inside the `record_to_sample()` function if that is the only place it is referenced.